app/assets/stylesheets/osem.css.scss

```diff
@@ -86,3 +86,51 @@ p.comment-body {
 #proposal-info div dt, #proposal-info div dd {
   display: inline-block;
 }
+
+.room-name{
```

Comment:
All of the files get merged into one large CSS file so all CSS is for everything. It's just a matter of better naming for humans :-)